volmdlr/edges.py

```diff
@@ -3605,19 +3583,6 @@ def rotation_inplace(self, center: volmdlr.Point3D, axis: volmdlr.Vector3D, angl
         self.curve = new_bsplinecurve3d.curve
         self.points = new_bsplinecurve3d.points
 
-    def translation(self, offset: volmdlr.Vector3D):
-        """
-        BSplineCurve3D translation
-        :param offset: translation vector
-        :return: A new translated BSplineCurve3D
-        """
-        new_control_points = [p.translation(offset) for p in
-                              self.control_points]
-        return BSplineCurve3D(self.degree, new_control_points,
-                              self.knot_multiplicities,
-                              self.knots, self.weights,
-                              self.periodic, self.name)
-
```

Comment:
Could You please verify if translation_inplace couldn't also be replaced? I see there's a little difference between the two, but I believe in principle they should be also the same...

Reply:
And maybe the same for rotation methods?

Reply:
@WirajanDASILVA It's done for `translation_inplace`